global.json

```diff
@@ -24,7 +24,7 @@
     "xcopy-msbuild": "17.8.5"
   },
   "native-tools": {
-    "jdk": "11.0.24"
+    "jdk": "latest"
```

Comment:
This would break https://github.com/dotnet/aspnetcore/blob/e0e0224d744e6219849538cf9ce5dd3db1ccd658/eng/scripts/InstallJdk.ps1#L24-L27 when we don't pass in a version - @BrennanConroy @halter73 maybe we could just default to https://aka.ms/download-jdk/microsoft-jdk-11.0.25-windows-x64.zip for now?

Reply:
Since we're likely changing the mechanism, let's just do whatever is needed to get this working.

Reply:
@ViktorHofer can you just change the block I linked to default to 11.0.24 for now?

Reply:
@wtgodbe feel free to push into this branch

Reply:
OK I think I got it right.